soc/ite/ec/it51xxx/chip_chipregs.h

```diff
@@ -298,4 +299,7 @@ struct gctrl_it51xxx_regs {
 /* 0x38: Special Control 9 */
 #define IT51XXX_GCTRL_ALTIE       BIT(4)
 
+/* Alias gpio_it8xxx2_regs to gpio_it51xxx_regs for compatibility */
+#define gpio_it8xxx2_regs gpio_it51xxx_regs
```

Comment:
this is a bit of a weird way of doing it, would it make sense to fix up the original driver to use a generic name?

Reply:
I have already updated the original related structures and registers to use a generic name, as seen in the commit: https://github.com/zephyrproject-rtos/zephyr/pull/87095/commits/77a54558980a8b4e5d081d7b85d29d85dccd6a28 .